cypress/plugins/index.js

```diff
@@ -17,11 +18,18 @@ module.exports = (on, config) => {
       auth0Password: process.env.AUTH0_PASSWORD,
       auth0Scope: process.env.AUTH0_SCOPE,
       auth0SessionCookieName: process.env.AUTH0_SESSION_COOKIE_NAME,
+      // Test Case Settings
       auth0Username: process.env.AUTH0_USERNAME,
       auth0UsernameAlt: process.env.AUTH0_USERNAMEALT,
       auth0PasswordAlt: process.env.AUTH0_PASSWORDALT,
+      // Cookie Settings
+      auth0CookiePath: process.env.AUTH0_COOKIE_PATH || '/',
```

Comment:
Does the addition of these options mean the `cypress.env.json` example in the README should be updated?

Reply:
Good catch!